README.md

```diff
@@ -208,7 +212,7 @@ Now we just need to add in the erb partials in `profile.erb` to display the data
         <div class="blankslate blankslate-narrow p-9">
             <img src="<%= user_photo_url %>" />
             <h3>
-                Hey, <a href="<%= user_url %>">@<%= user_login %></a>!
+                Hey, <a href="<%= user_login %>">@<%= user_login %></a>!
```

Comment:
I meant for this to link to the reader's github profile, so I'll fix this link to be `href="https://github.com/<%= user_login %>"`